library/known

```diff
@@ -0,0 +1,6 @@
+# maintainer: hello@withknown.com
+
+0.9.2: git://github.com/idno/Known-Docker@986a2618080f32bbbcb9af3c8e7c15297d9658ea
+0.9: git://github.com/idno/Known-Docker@986a2618080f32bbbcb9af3c8e7c15297d9658ea
+0: git://github.com/idno/Known-Docker@986a2618080f32bbbcb9af3c8e7c15297d9658ea
+latest: git://github.com/idno/Known-Docker@986a2618080f32bbbcb9af3c8e7c15297d9658ea
```

Comment:
I'd like this to be updated to the newer [RFC 2822 based format](https://github.com/docker-library/official-images/tree/a687a143dea0629a21364f2f3c2cadde595199dc#instruction-format) but this PR was made before that and it still works. @tianon will probably update it like rocket.chat and piwik anyway.